mule-deployer/src/main/java/org/mule/tools/client/standalone/installer/MuleStandaloneInstaller.java

```diff
@@ -129,10 +126,7 @@ protected File getDependency(ArtifactDescription artifactDescription)
       artifactResolver.resolve(artifact, mavenProject.getRemoteArtifactRepositories(), localRepository);
       return artifact.getFile();
     } catch (AbstractArtifactResolutionException e) {
-      throw new MojoExecutionException("Couldn't download artifact: " + e.getMessage(), e);
-    } catch (Exception e) {
-      throw new MojoFailureException("Couldn't download artifact: " + e.getMessage());
+      throw new DeploymentException("Couldn't download artifact: " + e.getMessage(), e);
```

Comment:
+      throw new DeploymentException("Couldn't download artifact: " + e.getMessage(), e);
vs 
+      throw new DeploymentException("Fail to download artifact: " + artifactDesdription.getName(), e);

Not a good idea to throw an exception with e.getMessage and then send also e as parameter

Reply:
Done! Thanks for the hint.